### PR TITLE
Added basic .travis.yml

### PR DESCRIPTION
**Pull request by sammyshj**
Added .travis.yml for Travis CI builds

### CODE REVIEW

**Comment by smcgregor** [comment]
Can you link me to this running on travis?

**Comment by sammyshj** [comment]
@smcgregor The builds can be seen at https://travis-ci.org/sammyshj/privly-safari/builds/64028422 and https://travis-ci.org/sammyshj/privly-safari/builds/64028541

**Comment by sammyshj** [comment]
I am currently working so that the tests pass and then I will open a new PR in privly/privly-safari repo which you can merge

**Comment by sammyshj** [comment]
I am looking at why the tests are failing now. Once the tests pass, you can review the code and I will squash the commits so that the PR can be merged

**Comment by sammyshj** [comment]
@smcgregor The build https://travis-ci.org/sammyshj/privly-safari/builds/64031717 fails where it cannot find the Safari binary. I think making the test run on SauceLabs would be a viable option.

**Comment by sammyshj** [comment]
@smcgregor The Travis CI build is successful now. It can be seen at https://travis-ci.org/sammyshj/privly-safari/builds/64033873
I have squashed all the commits and the PR can be merged now if the code looks good to you

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+language: node_js
+node_js:
+- "0.10"
+addons:
+  sauce_connect: true
+env:
+  global:
+  - TRAVIS_LAUNCHES_SAUCE_CONNECT=true
+  - secure: GljUxDK7oCkqvKWzP+OZUdx8lCFJ7FuEqxPH0Cmmijh21KKfvepI2tM6YtQ6dQ3Z8BNWLt5ckvQJNrTm/n49Q+cOJh3ulzesbu0/UdAFRWd0OW0hK9MZGnoTw3DF6NbcYDxeCCiYgq6bIj897RY4fKDszlibvMIS3BgQYMd8DFs=
```
Comment (smcgregor) [inline .travis.yml]: You should add comments here indicating what these will be when they are decrypted and which key they are associated with. At some point this file may have many different secure environmental variables.

**Comment by smcgregor** [comment]
Only one comment, otherwise this is ready to merge.

**Comment by sammyshj** [comment]
@smcgregor I have added the comment regarding SAUCE_USERNAME and SAUCE_ACCESS_KEY and squashed all the commits into one.

**Pull request merged**